store/details.go

```diff
@@ -41,6 +41,7 @@ type snapDetails struct {
 	Publisher        string             `json:"publisher,omitempty"`
 	RatingsAverage   float64            `json:"ratings_average,omitempty"`
 	Revision         int                `json:"revision"` // store revisions are ints starting at 1
+	ScreenshotURLs   []string           `json:"screenshot_urls,omitempty"`
```

Comment:
Having some store json of this in store_test.go would be good, e.g. MockDetailsJSON or MockSearchJSON. We will also need to update the curl line there as there is a new screenshot field there now.